website/docs/Publishing.md

```diff
@@ -46,4 +61,12 @@ Even if you are using Github Pages, and:
 
 ## Github Pages Custom Domain CNAME file
 
-Github Pages requires you to create a CNAME file in your website to host on a custom domain. You should create this file at `yourProjectRoot/.moonwave/static/CNAME` (create the folders if they don't exist) and put your domain in the file. 
\ No newline at end of file
+Github Pages requires you to create a CNAME file in your website to host on a custom domain. You should create this file at `yourProjectRoot/.moonwave/static/CNAME` (create the folders if they don't exist) and put your domain in the file. 
+
```

Comment:
I think it would make more sense to mention the `--out-dir` flag briefly [here](https://github.com/evaera/moonwave/blob/05e69f9854daab7f6d2811fcae3ce74095a9cafe/website/docs/Publishing.md?plain=1#L14C1-L14C126) instead of creating a new section for it. Just something saying, "Or you can specify `--out-dir [path]` to set the build directory to a different path`